modin/engines/base/io.py

```diff
@@ -273,42 +273,50 @@ def read_excel(
         io,
         sheet_name=0,
         header=0,
-        skiprows=None,
-        index_col=None,
         names=None,
+        index_col=None,
         usecols=None,
+        squeeze=False,
+        dtype=None,
+        engine=None,
+        converters=None,
+        true_values=None,
+        false_values=None,
+        skiprows=None,
+        nrows=None,
+        na_values=None,
         parse_dates=False,
         date_parser=None,
-        na_values=None,
         thousands=None,
+        comment=None,
+        skipfooter=0,
         convert_float=True,
-        converters=None,
-        dtype=None,
-        true_values=None,
-        false_values=None,
-        engine=None,
-        squeeze=False,
+        **kwds
     ):
         ErrorMessage.default_to_pandas()
         kwargs = {
             "io": io,
             "sheet_name": sheet_name,
             "header": header,
             "skiprows": skiprows,
+            "nrows": nrows,
             "index_col": index_col,
             "names": names,
             "usecols": usecols,
             "parse_dates": parse_dates,
             "date_parser": date_parser,
             "na_values": na_values,
             "thousands": thousands,
+            "comment": comment,
             "convert_float": convert_float,
             "converters": converters,
             "dtype": dtype,
             "true_values": true_values,
             "false_values": false_values,
             "engine": engine,
             "squeeze": squeeze,
+            "skipfooter": skipfooter,
+            "kwds": kwds,
```

Comment:
I think we need to do an `update` instead of setting `kwds=kwds` here. It will treat `kwds` as a keyword if we do it this way.

```python
kwargs = {...}
kwargs.update(kwds)
return cls.from_pandas(...)
```